docs/content/en/docs/installation/_index.en.md

```diff
@@ -118,6 +118,7 @@ $ helm install --name fission --namespace fission \
 $ export FISSION_NAMESPACE="fission"
 $ kubectl create namespace $FISSION_NAMESPACE
 $ helm install --namespace $FISSION_NAMESPACE --name-template fission \
+    --set serviceType=NodePort,routerServiceType=NodePort \
```

Comment:
Does minikube not by default expose these services on nodeport since it can provision load balancer?

Reply:
I did not try this on minikube. However this was more towards same `--set` params added for helm 2 but missing in helm 3 section. So I added for  helm3 section too.